contracts/gov/user-keeper/GovUserKeeper.sol

```diff
@@ -565,9 +565,10 @@ contract GovUserKeeper is IGovUserKeeper, OwnableUpgradeable, ERC721HolderUpgrad
     }
 
     function nftVotingPower(
-        uint256[] memory nftIds
+        uint256[] memory nftIds,
+        bool calculatePowerArray
```

Comment:
No need to have this parameter here, just hardcode `true` please

Reply:
done